actor/builtin/storagemarket/storagemarket.go

```diff
@@ -193,6 +197,53 @@ func (sma *Actor) UpdateStorage(vmctx exec.VMContext, delta *types.BytesAmount)
 	return 0, nil
 }
 
+func (sma *Actor) GetSlashableMiners(vmctx exec.VMContext) (*[]address.Address, uint8, error) {
```

Comment:
Same comment about naming as above propagates to here.  Consider `GetProvingMiners`

Reply:
So I had thought about this -- I was allowing for the possibility that other factors besides nonzero proving set may be used to determine slashability.  I'm not married to the name though.